src/modules/Dashboard/features/Applet/Popups/TransferOwnershipPopup/TransferOwnershipPopup.test.tsx

```diff
@@ -0,0 +1,63 @@
+import { fireEvent, waitFor, screen } from '@testing-library/react';
+import axios from 'axios';
+
+import { renderWithProviders } from 'shared/utils/renderWithProviders';
+import { Applet } from 'api';
+
+import { TransferOwnershipPopup } from '.';
+
+const email = 'test@gmail.com';
+const mockedApplet = {
```

Comment:
maybe move mocked applet to shared mock consts for tests to use it in other tests as well as a test email?

Reply:
fixed